no-literal-securerandom-exact-cmd.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+## This script runs wpi-many.sh with appropriate arguments and
+## environment variables. It is copied from the experiments for the
+## Continuous Compliance paper. If you want to re-use this script,
+## you'll have to change environment variables and paths below.  Note
+## that this script is intended for use with a custom typechecker
+## (i.e., a typechecker that is not in the main Checker Framework
+## distribution).  If your typechecker is in the main Checker
+## Framework distribution, you should use wpi-many.sh or
+## wpi.sh directly rather than making a copy of
+## this script.
+
+## Change these if necessary.
+
+export JAVA11_HOME=/usr/lib/jvm/java-11-openjdk/
+export JAVA8_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+
+## Everyone must change these.
+
+export PARENTDIR=${HOME}/compliance-experiments/fse20
+export CHECKERFRAMEWORK=${PARENTDIR}/checker-framework
+checker=org.checkerframework.checker.noliteral.NoLiteralChecker
+checkername=no-literal
+repolist=securerandom.list
+
+# the stub files for the checker being used
```

Comment:
It seems you have specified a directory rather than a file.  Please clarify the comment or point at do-like-javac's documentation of the `--stubs` flag.